.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.1.0a
```

Comment:
We don't have to change this with each release. It's just a marker for bumpversion to do its thing during the release process. We could keep it at 1.0.0a at all times. Changing this string is error-prone. If we miss a file where this placeholder string is referenced, it will produce a faulty release. It's ok to change it to 1.2.0a in this release, but I recommend we go back to 1.0.0a in the next release and leave it alone :).

Reply:
Sure Adam, We will revert this change in the future release.
Thanks for pointing this.